lib/core/src/Cardano/Wallet.hs

```diff
@@ -976,14 +970,14 @@ signDelegation ctx wid argGenChange pwd coinSel poolId action = db & \DBLayer{..
             withExceptT ErrSignDelegationNoSuchWallet $
                 putCheckpoint (PrimaryKey wid) (updateState s' cp)
 
-            let rewardAccount = deriveRewardAccount @k pwd xprv
+            let rewardAcc = deriveRewardAccount @k pwd xprv
```

Comment:
This is a private key. Maybe we should rename `HasRewardAccount.rewardAccount` to `rewardAccountXPub` or something.

Reply:
`rewardAccountXPub` is redundant with the type signature. Instead, we could make `deriveRewardAccount` part of the `HasRewardInterface` to make sure they're done consistently. Pretty minor here however.